test/charts/c3/c3.spec.js

```diff
@@ -0,0 +1,25 @@
+describe('Directive: c3Chart', function() {
```

Comment:
Shouldn't the name of this file match the name of the directive? I would think this should be named `c3-chart.spec.js`.